pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -66,6 +67,17 @@ func createBlobAccess(configuration *pb.BlobAccessConfiguration, storageType str
 		return nil, errors.New("Storage configuration not specified")
 	}
 	switch backend := configuration.Backend.(type) {
+	case *pb.BlobAccessConfiguration_Badger:
+		backendType = "badger"
+		opts := badger.DefaultOptions
+		opts.Dir = backend.Badger.Directory
```

Comment:
Why not use `badger.DefaultOptions(backend.Badger.Directory)` ? According to [documentation](https://godoc.org/github.com/dgraph-io/badger#Options.WithDir) this sets both `Dir` and `ValueDir` to the provided path.